infrastructure/pyd/references.d

```diff
@@ -218,10 +218,14 @@ struct DClass_Py_Mapping {
 /// inside a destructor and we need to use this container there.
 template reference_container(Mapping) {
     static if(is(Mapping == DStruct_Py_Mapping)) {
-        alias MultiIndexContainer!(Mapping, IndexedBy!(
-                    HashedNonUnique!("a.d"), "d",
-                    HashedUnique!("a.py"), "python"),
-                MallocAllocator, MutableView)
+        // See #104 for the reason for the hash functions below
+        alias MultiIndexContainer!(
+            Mapping,
+            IndexedBy!(
+                HashedUnique!("a.d"), "d",
```

Comment:
NonUnique

Reply:
Ah, ok, I don't know how I missed it was a different type.